school_lunch/i18n/fr_BE.po

```diff
@@ -31,8 +31,8 @@ msgstr ""
 
 #. module: school_lunch
 #: model_terms:ir.ui.view,arch_db:school_lunch.menu_form
-msgid ", week"
-msgstr ", semaine"
+msgid "<span class=\"oe_inline\">, week </span>"
+msgstr "<span class=\"oe_inline\">, semaine </span>"
```

Comment:
As long as it works okay, but as advice, keeping translations updated manually can be hard, it will be easier to export the module translation from the UI and update any terms missing (possibly with a PO editor)